test/core/BehaviorTree-Serialization.js

```diff
@@ -111,7 +110,7 @@ suite('Core: Behavior Tree - Serialization', function() {
 
     test('Load JSON model with custom nodes', function() {
         var tree = new BehaviorTree();
-        var CustomNode = Class(Condition);
+        var CustomNode = Condition;
```

Comment:
can we remove this line and replace all the occurrences of `CustomNode` with `Condition`?